### PR TITLE
fix: ignore empty attributes string

### PR DESCRIPTION
**Pull request by akoshman**
Hi, there is a fix for lines without attributes specified.
And a small improvement to ignore blank lines

### CODE REVIEW

**Comment by Raiper34** [comment]
Hello friend, thank you for your PR, I see, you added some new branches and statements, but they are not unit tested, can you add unit tests, please? You can check coverage when you run `npm run test` and then you can see it in *coverage/lcov-report/index.ts* file in your browser. Can you cover `m3u-parser.ts` to keep 100% coverage, please? After that I will merge it and release ;) Best!

File: src/m3u-parser.ts [hunk 1]
```diff
@@ -18,11 +18,14 @@ export class M3uParser {
...
-    const attributeValuePair = attributesString.split('" ');
     const attributes: M3uAttributes = new M3uAttributes();
+    if (!attributesString) {
+      return attributes;
+    }
+    const attributeValuePair = attributesString.split('" ');
     attributeValuePair.forEach((item) => {
       const [key, value] = item.split('="');
-      attributes[key] = value.replace('"', '');
+      attributes[key] = value ? value.replace('"', '') : value;
```
Comment (ducktrshessami) [inline src/m3u-parser.ts:28]: This new branch seems to only run if an attribute is improperly formatted

**Comment by ducktrshessami** [comment]
I started writing tests to PR to this fork, but noticed the above. Since the author of this PR seems to be inactive, how should I proceed?

**Comment by Raiper34** [comment]
> I started writing tests to PR to this fork, but noticed the above. Since the author of this PR seems to be inactive, how should I proceed?

Any progress with that? Do you need help from my side? I would add an improperly formatted attribute unit test if we want to keep this change.

**Comment by ducktrshessami** [comment]
Forgot to mention for link akoshman/m3u-parser-generator#1.
I figured it could be left in, but should error instead of inserting an `undefined` value. A test was written as well.
Just waiting on @akoshman.

**Comment by akoshman** [comment]
Hello @ducktrshessami 
You can just take my commit and make a new pull request.

**Comment by ducktrshessami** [comment]
Uh alright

**Comment by Raiper34** [comment]
Finished in #5

**Pull request closed without merging**